foxpuppet/windows/browser/tabbar.py

```diff
@@ -0,0 +1,46 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from foxpuppet.windows.browser.tab import Tab
+
+
+class TabBar(Tab):
```

Comment:
The `TabBar` class should have nothing to do with the `Tab` class. It only contains `Tab` instances as properties. Also this is a `Region` too, right?